src/checkbox/examples.js

```diff
@@ -263,7 +263,7 @@ export default {
             }
           }}
         />
-        <Checkbox onChange={onChange} inputRef={inputRef} autoFocus={isFocused}>
+        <Checkbox onChange={onChange} inputRef={inputRef}>
```

Comment:
so after removing `autoFocus` it doesnt' work as expected in this example

Reply:
The "Click to focus on checkbox" still seems to be working. The focus should be handled by the `inputRef` since autoFocus only happens on initial render.

Reply:
does this piece still work?
https://github.com/uber-web/baseui/blob/master/src/checkbox/checkbox.js#L47

Reply:
It still works, but it's an accessibility anti-pattern. It's also a native `<input />` attribute so not sure what the custom implementation is for 🤷‍♂️

Reply:
We should still support the `autoFocus` case, that sets focus on the initial render.